client/src/PublicHome.js

```diff
@@ -1,26 +1,75 @@
 import React, { Component } from 'react';
 import authorizeUser from './Auth'
 import { Redirect } from 'react-router';
+import { Link } from 'react-router-dom';
 import { connect } from 'react-redux'
 import Loading from './components/Loading'
-import { Button, Jumbotron, Media, Badge } from 'reactstrap';
-import Login from './img/Login.png';
-import LoginHover from './img/LoginHover.png';
-import Register from './img/Register.png';
-import RegisterHover from './img/RegisterHover.png';
-import Logo from './img/Logo.png';
-import Houses from './img/HousesMedium.png';
+import { Button, Jumbotron, Media, Badge, Fade,
+Modal, ModalBody, ModalHeader } from 'reactstrap';
+import Register from './Register';
+import Login from './Login';
+import LoginImg from './img/Login.png';
+import LoginHoverImg from './img/LoginHover.png';
+import RegisterImg from './img/Register.png';
+import RegisterHoverImg from './img/RegisterHover.png';
+import RegisterAdminImg from './img/RegisterAdmin.png';
+import RegisterRegularImg from './img/RegisterRegular.png';
+import GroupsImg from './img/Groups.png';
+import GroupsHoverImg from './img/GroupsHover.png';
+import LogoImg from './img/Logo.png';
+import LogoHoverImg from './img/LogoHover.png';
+import HousesImg from './img/HousesMedium.png';
+import HousesHoverImg from './img/HousesMediumHover.png';
 import { AiTwotoneHome } from 'react-icons/ai';
 
 class PublicHome extends Component {
   constructor(props){
     super(props);
     this.state = {
       login_hover: false,
-      regiser_hover: false,
+      register_hover: false,
+      houses_hover: false,
       show_info: false,
-      show_resident_info: false
-    }
+      show_resident_info: false,
+      show_register_info: false,
+      logo_hover: false,
+      modal: false,
+      type: false,
+      typeText: "login"
+    };
+    this.toggleRegister = this.toggleRegister.bind(this);
+    this.toggle = this.toggle.bind(this);
+    this.toggleLogin = this.toggleLogin.bind(this);
+    this.toggleRegAdmin = this.toggleRegAdmin.bind(this);
+    this.toggleRegRegular = this.toggleRegRegular.bind(this);
+  }
+
+  toggle = () => this.setState({modal: !this.state.modal});
+  toggleLogin = () => {

```

Comment:
Same functionality for Login/Register modal appearing like in navbar, but for main public homepage buttons